policies/instance-kms-policy.json

```diff
@@ -0,0 +1,18 @@
+{
+    "Version": "2012-10-17",
+    "Statement": [
+        {
+            "Action": [
+                "kms:Encrypt*",
+                "kms:Decrypt*",
+                "kms:ReEncrypt*",
+                "kms:GenerateDataKey*",
+                "kms:Describe*"
```

Comment:
suggestion: I think `Decrypt` and `GenerateDataKey` should be sufficient according to [AWS docs](https://docs.aws.amazon.com/AmazonS3/latest/userguide/UsingKMSEncryption.html). Could you please check this?

Reply:
Updated the policy to only include `Decrypt` and `GenerateDataKey`. Validated that the change is working correctly. 

If the `SecureString` SSM parameters are ever updated to use the CMK, `kms:Encrypt` will have to be added to this policy.